Source/fontconfig/FCFontEnumerator.m

```diff
@@ -214,75 +420,86 @@ - (void) enumerateFontsAndFamilies
   allFontNames = fcxft_allFontNames;
   allFontFamilies = fcxft_allFontFamilies;
   __allFonts = fcxft_allFonts;
+
+  // Sort font families
+  {
+	NSComparisonResult (*fontSort) (id, id, void *) = sortFontFacesArray;
+    NSEnumerator *e = [allFontFamilies keyEnumerator];
+    id key;
+	while ((key = [e nextObject]))
+	  {
+		[[allFontFamilies objectForKey: key] sortUsingFunction:fontSort context:NULL];
+	  }
+  }
 }
 
 - (NSString *) defaultSystemFontName
 {
-  if ([allFontNames containsObject: @"Bitstream Vera Sans"])
+  if ([allFontNames containsObject: @"DejaVuSans"])
```

Comment:
Nothing against making DejaVu the default font, but I will have to test whether the names match the new results for "name" on my machine and hopefully many others :-)

Reply:
DejaVu is Vera Sans but with many more glyphs, so anywhere the default font is Vera should really prefer DejaVu _if present_, falling back to Vera and then finally to something that looks worse. :)